changelogs/fragments/fix-oc-conn-plugin-envvar.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
```

Comment:
```suggestion
breaking_changes:
```

Reply:
Force pushed and lost this, But I caught it... This change should be restored.